### PR TITLE
Create Login endpoint & Signup Endpoint

### LINKED ISSUE #17: Create Login endpoint
POST `/login/user`

BODY: 
```
{ 
     email: string,
     password: string
}
```

Response: 
```
{
    first_name: string
    last_name: string
    id: string (this is the UUID that will be used as URL identifier)
    available_points: int
    created_date: month year
    currently_reading: {
        title: string
        author: string
        category: string
   }
}
```

Depends on #16

### PR DESCRIPTION
**Pull request by tosintubi**
This PR fixes #17 and #16. 
It includes functionality for user login and signup endpoints as well as test cases covering them

### CODE REVIEW

**Review by delphine-boke — DISMISSED** [review]
please separate the login and signup endpoints as one branch

**Comment by kosyfrances** [comment]
@tosintubi is this ready for a review now?

**Comment by tosintubi** [comment]
> 

@kosyfrances Yes, it is ready for review

File: README.md [hunk 1]
```diff
@@ -217,6 +217,12 @@ Please install and set up the following packages first. Upgrade if you find the
   flask create_tables
   ```
 
+- Seed data generation. This will generate some data on your local postgresql database for testing purposes. The argument `NUMBER` indicates the amount of seed data to be generated. `NUMBER = 1` will generate 2 User profiles (1 Borrower and 1 Lender),  2 Books (the first Book is borrowed, while second Book is available for loan).
+This command will also generate seed data on your heroku database if you execute this command on heroku's console.
+Note: You will need to create the database and update the `DATABASE_URL` entry in your `.env` file.
+  ```sh
+  flask generate_data [NUMBER]
+  ```
```
Comment (kosyfrances) [inline README.md:225]: I remember this is from a different PR. How is it magically here as a new change? 🤔
You probably need to rebase this branch properly.

**Review by kosyfrances — CHANGES_REQUESTED** [review]
Please rebase this branch properly as the seeder task should not be here as a new change.
I'll review when things are properly cleaned up.

**Pull request closed without merging**